src/pages/FavouritesPage/Favourites.tsx

```diff
@@ -25,7 +25,7 @@ export const Favourites: React.FC<Props> = ({
 
         <div className="favourites__content">
           <h1 className="favourites__title">
```

Comment:
filename and all classes are still with typo
You may do this in VSCode (FYI):
![image](https://github.com/mate-academy/react_phone-catalog/assets/86923089/348388d7-da1f-4442-911b-eb8be5668be4)